ckanext/publications_qld_theme/plugin.py

```diff
@@ -103,6 +114,22 @@ def populate_revision(resource):
         resource['revision_timestamp'] = current_revision.revision_timestamp
 
 
+def unreplied_comments_x_days(thread_url):
+    """A helper function for Data.Qld Engagement Reporting to highlight un-replied comments
+    after x number of days (number of days is a constant in the reporting plugin).
+    """
+    comment_ids = []
+
+    if 'data_qld_reporting' in config.get('ckan.plugins', False):
```

Comment:
is this needed as we are removing reporting, or is your goal to have both use same theme?

Reply:
I don't think they'll ever be able to use the same theme, but I'd like to minimise the differences. Since this check has robustness built in, I kept it.